packages/documentation/src/stories/foundations/palettes/palettes.docs.mdx

```diff
@@ -25,3 +26,5 @@ You can apply this text color to any element by using the `.palette-text` class.
 <div className="hide-col-default">
   <Controls of={paletteStories.Default} />
 </div>
+
+<StylesPackageImport components={['post-palettes']} />
```

Comment:
It would be good if the import could adapt to the selected theme. It is stored in the local storage under `swisspost-documentation-theme`, if not defined the default is "Post".

Reply:
Would it also update `post-palettes` for `cargo-palettes`?

Reply:
Yes, if you check here and switch the theme, you'll see it changes it dynamically to cargo-palettes: https://preview-4127--swisspost-design-system-next.netlify.app/?path=/docs/43481535-5b39-40b5-a273-478b07dc3b31--docs